### DISCUSSION TITLE
Slice/clip a geometry using TRame

### DISCUSSION THREAD

**Discussion by indominus-hub** [Kitware/trame#74] [General]
Dear all,

I was wondering if Trame is able to slice/clip an .vtu file using associated DICOM images dataset. Kindof slice/clip by plane of Paraview, but using DICOM planes (axial, saggital and coronal).

Any ideas?

Many thanks in advance!

**Comment by jourdain** [comment]
Yes trame can do anything that VTK or ParaView can do. You just need to have at least VTK on your server side. You can even add ITK too for reading your DICOM.

**Reply by indominus-hub** [reply]
Thanks Jourdain!

Sorry for my late answer. We are working on that!

Congratulations for such a tool! It looks impressive  and full of interesting capabilities.

Can't wait to test it displaying my results through a web connection (now playing with localhost only)

**Comment by jourdain** [comment]
For multi-user setup you could use the docker setup that the cookie-cutter provide you with. And for [remote rendering](https://github.com/Kitware/paraview-visualizer/blob/master/docker/Dockerfile#L1) and/or [ParaView integration](https://github.com/Kitware/paraview-visualizer/blob/master/docker/Dockerfile#L8-L10), you can look at how we did it with [Visualizer](https://github.com/Kitware/paraview-visualizer)
